### PR TITLE
XML methods for remaining commands

### PR DESCRIPTION
**Pull request by davemevans**
Note that PrivateDescriptor doesn't work correctly due to an existing issue (see #112) in the decode method with the command length being calculated incorrectly.

I also changed the length of the identifier field to 32bits to match the specification (See Table 13)

### CODE REVIEW

File: threefive/commands.py [hunk 1]
```diff
@@ -37,6 +37,11 @@ def encode(self, nbin=None):
         nbin = self._chk_nbin(nbin)
         return nbin.bites
 
+    def from_xml(self, stuff):
```
Comment (futzu) [inline threefive/commands.py:40]: this should be base.py SCTE35Base class. It has the xml()

**Review by futzu — COMMENTED** [review]
Does this fix the command length in Private Command?

File: threefive/commands.py (hunk 1 above)
Comment (davemevans) [reply threefive/commands.py:40]: Yeah, fair point - I should have spotted that. I'd have removed this but you already merged it :smile:

**Comment by davemevans** [comment]
> Does this fix the command length in Private Command?

It doesn't sadly.

**Pull request merged**